conversion/indico_conversion/conversion.py

```diff
@@ -131,3 +200,28 @@ def _process(self):
                     continue
                 containers[attachment.id] = tpl.render_attachments_folders(item=attachment.folder.object)
         return jsonify(finished=finished, pending=pending, containers=containers)
+
+
+@celery.periodic_task(bind=True, max_retries=None, run_every=crontab(minute='0', hour='2'))
+def check_cloudconvert_credits(task):
+    from indico_conversion.plugin import ConversionPlugin
+
+    api_key = ConversionPlugin.settings.get('cloudconvert_api_key')
+    client = CloudConvertRestClient(api_key=api_key, sandbox=False)
+    notify_threshold = ConversionPlugin.settings.get('notify_threshold')
+    notify_email = ConversionPlugin.settings.get('notify_email')
+
+    try:
+        credits = client.get_remaining_credits()
+    except requests.RequestException as err:
+        ConversionPlugin.logger.info('Could not fetch the remaining CloudConvert credits: %s', err)
+        task.retry(countdown=60)
+
+    if notify_threshold is not None and credits <= notify_threshold:
```

Comment:
Does it make sense to even run the task (and send the request) when the threshold is None?

Reply:
good point, I'd say no